src/test/Recordings/Filters/DateTimePicker.test.tsx

```diff
@@ -46,6 +46,11 @@ const onDateTimeSelect = jest.fn((date) => {
 });
 const currentDate = new Date('14 Sep 2022 00:00:00 UTC');
 
+// See https://github.com/testing-library/user-event/issues/833 with new testing-library versions
```

Comment:
Going by https://github.com/testing-library/user-event/issues/833#issuecomment-1171452841 , would it be better to use a mock test-controlled timer rather than disabling the delays altogether?

https://jestjs.io/docs/timer-mocks